packages/ketcher-react/src/script/ui/state/options/index.js

```diff
@@ -81,6 +81,12 @@ export const initOptionsState = {
 };
 
 function getSerilizedServerOptions(options) {
+  const renderStereoStyle = !options.showStereoFlags
+    ? 'none'
+    : options.ignoreChiralFlag
+    ? 'ext'
+    : 'old';
```

Comment:
We're currently working on refactoring these nested ternary operators. Could you please rewrite it to the if-else statements sequence?